jmix-templates/content/flowui/list/controller.java

```diff
@@ -16,5 +27,19 @@
 @ViewDescriptor("${viewDescriptorName}.xml")
 @LookupComponent("${tableId}")
 @DialogMode(width = "64em")
-public class ${viewControllerName} extends StandardListView<${entity.className}> {
+public class ${viewControllerName} extends StandardListView<${entity.className}> {<%if (useDataRepositories){%>
+
+    @Autowired
+    private ${repository.getQualifiedName()} repository;
+
```

Comment:
Two new lines between autowired repository and the next method. Should be one, I guess.
Check the same in other templates. There should be one empty line between generated fields and methods. Otherwise, the generated controller looks a bit untidy.